site/_posts/2020-06-03-user-registration-and-sign-in-with-laravel.md

```diff
@@ -257,11 +292,26 @@ class LoginUser extends Controller
     return redirect('profile');
   }
 }
+//..
+```
+
+Finally, this will require two new routes, so open your `routes/web.php` file again and add the following lines:
+
+```php
+Route::view('/', 'login');
+Route::post('/login', 'LoginUser');
 ```
 
-_Note: When users are logged in, FusionAuth also returns a JWT token. If you're building a single page application or want to perform client-side requests, [check out the documentation](https://fusionauth.io/docs/v1/tech/apis/jwt) for details._
+The first route will display a new view file that includes an email and password input field. The second route will use a new action controller to log a user in via FusionAuth. Your login page should look like this:
+
+{% include _image.liquid src="/assets/img/blogs/fusionauth-laravel/login-page.png" alt="The profile page for a Laravel application using FusionAuth." class="img-fluid bottom-cropped" figure=false %}
+
+{% include _callout-tip.liquid content="When users are logged in, FusionAuth also returns a JWT token. If you're building a single page application or want to perform client-side requests, [check out the documentation](https://fusionauth.io/docs/v1/tech/apis/jwt) for details."
+%}
+
+Now when you login using the new form, you'll be redirected to `/profile`, but that page doesn't exist yet (hence, the 404 error)
```

Comment:
Missing a period on the end of this sentence.

Reply:
the `period` is `404` just like our profile page ... lol 🤣 
...and be happy that my calling was not comedy 😄